src/components/home.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import JSONTree from 'react-json-tree';
+
+import DivJSON from './div_json';
+
+import { createAuth, getTokens, renewIdToken, decodeIdToken, refreshSession, closeSession, signOut } from '../actions/index';
+
+class Home extends Component {
+  componentWillMount() {
+    //  create auth that goes in props
+    this.props.createAuth();
+  }
+
+  sessionClosed() {
+    return localStorage.getItem('session_closed');
```

Comment:
Why is this in localStorage? Shouldn't this be something we get from our state/store?